public/views/signin.html

```diff
@@ -44,7 +44,7 @@ <h2 id="slogan">Sign in with:</h2>
           <div class="sign-error" ng-show="showError() === 'invalid'">
             Please check your username/password and try again.
           </div>
-          <form class="signup form-horizontal" action="/users/session" method="post">
+          <form class="signup form-horizontal" action="/api/auth/login" method="post">
```

Comment:
this form should no longer use the post action, but call a function in the controller, see @andela-foladipo for clarity